types/mangopay2-nodejs-sdk/mangopay2-nodejs-sdk-tests.ts

```diff
@@ -0,0 +1,721 @@
+import * as Mangopay from "mangopay2-nodejs-sdk";
```

Comment:
Since the export is commonjs-style `export = Mangopay`, please use commonjs-style `import Mangopay = require("mangopay2-nodejs-sdk")`

Reply:
Fixed in the new PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32050